Packs/CommonScripts/ReleaseNotes/1_7_7.md

```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+##### GenerateRandomString
+- Improved implementation with limiting the maximum length of the generated string to be 10,000.
```

Comment:
```suggestion
- The max supported *Length* is now up to 10,000 characters. If *Length* of more than 10,000 is provided, it'll be set to 10,000.
```

Reply:
Done, changed the first sentence you suggested to `You can now use *Length* only up to 10,000 characters.` to match our available prefixes.